docs/pages/guides/typescript.mdx

```diff
@@ -0,0 +1,83 @@
+import PageMeta from '_shared/PageMeta';
+
+<PageMeta
+  title="Typescript – @material-ui/pickers"
+  description="A guide of how to use @material-ui/pickers with typescript"
+/>
+
+## TypeScript guide
```

Comment:
```suggestion
# TypeScript guide
```

Reply:
Ohhh no no no, our styling is really weird if using right h1, h2 :) 

Needs to be fixed or wait for migration

Reply:
Ok, let's wait migration :)